lib/AST/Parameter.cpp

```diff
@@ -102,8 +103,12 @@ ParameterList *ParameterList::clone(const ASTContext &C,
       decl->setName(C.getIdentifier("argument"));
     
     // If we're inheriting a default argument, mark it as such.
-    if (hadDefaultArgument && (options & Inherited)) {
-      decl->setDefaultArgumentKind(DefaultArgumentKind::Inherited);
+    // FIXME: Figure out how to clone default arguments as well.
```

Comment:
I had that at first and then had a wave of paranoia that someone would be depending on the current behavior. Not copying over the expression is *nearly* the same as setting the kind to None, but not quite.